### DISCUSSION TITLE
Command "react-scripts build" exited with 127

### DISCUSSION THREAD

**Discussion by giorgio40** [vercel/next.js#40733] [Help]
On my local machine my create-react app runs fine. I can npm start and dont get an error when I try to deploy I get this: Command "react-scripts build" exited with 127 whats going on here?

**Comment by icyJoseph** [comment]
Not sure this is the right repository to ask this question but I think your app is not at the root level of the repository, so you have to tell Vercel where the application lives, https://vercel.com/docs/concepts/deployments/configure-a-build#root-directory.

In your case I think it should be `./ketodogapp`, and that should do the trick.

**Reply by shahad-hassan19** [reply]
Thank you so much!

**Reply by MementoMori2323** [reply]
Did it worked for you?

**Reply by shahad-hassan19** [reply]
yeah, my project structure was like this: 
my-app 
├── frontend 
└── server
so when deploying frontend, root directory should be frontend, and when deploying server, root directory should be server.
Root directory should be parent dir of main file(Entry file of project, like index.js for server and index.html for frontend).
